diff --git a/opigno_learning_path/src/Plugin/views/filter/OpignoGroupMembershipBoolean.php b/opigno_learning_path/src/Plugin/views/filter/OpignoGroupMembershipBoolean.php
index 0674cbd..b2e52c7 100644
--- a/opigno_learning_path/src/Plugin/views/filter/OpignoGroupMembershipBoolean.php
+++ b/opigno_learning_path/src/Plugin/views/filter/OpignoGroupMembershipBoolean.php
@@ -19,7 +19,7 @@ class OpignoGroupMembershipBoolean extends BooleanOperator {
   /**
    * {@inheritdoc}
    */
-  protected function operators() {
+  public function operators() {
     // We don't need any operators.
     return [];
   }
-- 
2.37.1.windows.1

